java/gast_to_code/gast_to_code_java.py

```diff
@@ -27,7 +27,20 @@ def handle_log_statement(gast):
         return "System.out.println"
 
     def handle_var_assign(gast):
-        pass
+        var_id = router.gast_to_code(gast["varId"], "js")
+        var_value = router.gast_to_code(gast["varValue"], "js")
+        var_type = gast["varValue"]["type"]
+        
+        if var_type == "num":
+            kind = "int"
+        elif var_type == "str":
+            kind = "String"
+        elif var_type == "bool":
+            kind = "boolean"
+        else:
+            kind = "customType"
+
+        return kind + " " + var_id + " = " + var_value
```

Comment:
Would this be the right place to add semicolons?

Reply:
+1 not sure how semicolons are working.